src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_metadata.py

```diff
@@ -337,7 +335,7 @@ def generate_channel(recurse=False, status=NEW):
         # Remove the channel and read it back from disk
         for c in chan.contents:
             c.delete()
-        my_dir = path_util.abspath(self.mds.ChannelMetadata._channels_dir / chan.dirname)
+        my_dir = pathlib.Path(self.mds.ChannelMetadata._channels_dir, chan.dirname).resolve()
```

Comment:
Is `resolve` really necessary? `/` syntax?

Reply:
Removed it.